src/BakePlugin.php

```diff
@@ -21,34 +21,30 @@
 use Cake\Console\CommandCollection;
 use Cake\Core\BasePlugin;
 use Cake\Core\Configure;
-use Cake\Core\Plugin as CorePlugin;
+use Cake\Core\Plugin;
 use Cake\Core\PluginApplicationInterface;
-use Cake\Routing\RouteBuilder;
 use DirectoryIterator;
 use ReflectionClass;
 use ReflectionException;
 
 /**
  * Plugin class for bake
  */
-class Plugin extends BasePlugin
+class BakePlugin extends BasePlugin
 {
     /**
      * Plugin name.
      *
      * @var string
```

Comment:
```suggestion
     * @var string|null
```
if you meant for this to be nullable

Reply:
I wish we didn't have to add these redundant docblock types anymore :(.

Reply:
We need to do a test with the slevomat setting that detects redundant blocks. I don't know if it forces all @param annotations to be removed even if there's a description you want to keep. Should we just not have comments on parameters annotations anymore?

Reply:
I don't think we should drop parameter comments but I doubt it would allow param description without the type.

Reply:
Need to test if a comment prevents it from being marked useless.

Reply:
I would rather check why the sniffs are missing for it:
https://github.com/spryker/code-sniffer/blob/master/Spryker/Sniffs/Commenting/DocBlockReturnNullSniff.php
or
https://github.com/spryker/code-sniffer/blob/master/Spryker/Sniffs/Commenting/DocBlockReturnNullableTypeSniff.php
could be the right direction here.

As long as they are properly synced, which is easier, all is well.